README.md

```diff
@@ -1060,6 +1064,100 @@ A targeted notification response will be returned as an XML representation, prov
 </targetedNotificationResponse>
 ```
 
+### Stored File
+
+A stored file represents a file which is associated with a node and which is stored in AWS S3.
+
+#### XML Representation
+
+##### Single Supported File Usage
+
+``` xml
+<supportedFileUsage name="SOME_FILE_USAGE">
+    <fileTypes>
+        <fileType>.someExtension</fileType>
+        <fileType>.someOtherExtension</fileType>
```

Comment:
Nitpick - these are file extensions (and what's with the leading `.`?) which don't really map that nicely onto actual file types.

Reply:
@rmanthorpe Not ideal, I agree, but it's consistent with the Publisher UI and usage to file type/extension association.